Client/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This is just the default [basic configuration](https://prettier.io/docs/en/configuration.html#basic-configuration) for prettier. We can get as specific here as we want. At hubspot, we surveyed all of our frontend devs to arrive at a shared prettier config that we're all comfortable with.